plugin/ochttp/server.go

```diff
@@ -70,6 +70,12 @@ type Handler struct {
 	// from the information found in the incoming HTTP Request. By default the
 	// name equals the URL Path.
 	FormatSpanName func(*http.Request) string
+
+	// IsHealthEndpoint holds the function to use for determining if the
+	// incoming HTTP request should be considered a health check. This is in
+	// addition to the private isHealthEndpoint func which may also indicate
+	// tracing should be skipped.
+	IsHealthEndpoint func(string) bool
```

Comment:
Description looks good now.

It might be better to pass http.Request rather than just Path. For example, decision could be made on a header.

Reply:
Makes perfect sense since the other funcs also accept the `http.Request`, I feel silly for not having considered it sooner. Thanks for the suggestion!